include/deal.II/fe/fe_q_base.h

```diff
@@ -324,6 +324,12 @@ class FE_Q_Base : public FE_Poly<POLY,dim,spacedim>
    * Mutex for protecting initialization of restriction and embedding matrix.
    */
   mutable Threads::Mutex mutex;
+
+  /*
+   * For the FE_Q_Bubbles class the highest polynomial degree is
+   * greater than the degree of FE_Q by one.
+   */
```

Comment:
The degree stored in FiniteElementData::degree is defined as the highest degree of a one dimensional polynomial needed for your space. This is what should be set in there, because the automatic selection of quadrature rules depends on it. So, I am not sure how to read your comment, but q_degree here should not be greater.